tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -123,10 +123,39 @@ export class CardFobControllerComponent {
     return this.axisDirection === AxisDirection.VERTICAL;
   }
 
+  private moveCurrentFob(
```

Comment:
This name is confusing as you may actually be moving the other fob. Also, it does not actually move the fob it just calculates the new timeSelection. I think we should do 1 of 2 things. The easiest it probably to just rename this to something like getNewTimeSelection. However, I also think it might be a reasonable idea to just have this function emit the event when the time is right.

Reply:
Yeah... this name is definitely a doozy, I'll change it to `getNewTimeSelection`. I'm trying to break the calculation apart from the event emitter.